client/app/domain-operation/progress/domain-operation-progress.html

```diff
@@ -94,8 +94,8 @@ <h3 class="mb-0" data-ng-bind=":: 'domains_operations_progress_step_' + ctrlDoma
                         </ul>
                         <ul data-ng-if="ctrlDomainOperationProgress.currentStepIndex === 1">
                             <li data-translate="domains_operations_progress_instructions_3"></li>
-                            <li data-translate="domains_operations_progress_instructions_4"></li>
-                            <li data-translate="domains_operations_progress_instructions_5"></li>
+                            <li data-translate="{{ ctrlDomainOperationProgress.domainExtension === 'fr' ? 'domains_operations_progress_instructions_fr_4' : 'domains_operations_progress_instructions_4' }}"></li>
```

Comment:
```suggestion
                            <li data-translate="{{:: ctrlDomainOperationProgress.domainExtension === 'fr' ? 'domains_operations_progress_instructions_fr_4' : 'domains_operations_progress_instructions_4' }}"></li>
```
Also, `'fr'` should be a constant

Reply:
The one-time binding is done. Having 'fr' as a constant is handled in another way now :)